Gopkg.toml

```diff
@@ -43,35 +43,35 @@ required = [
 
 [[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.11.3"
+  version = "kubernetes-1.12.5"
 
 [[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.11.3"
+  version = "kubernetes-1.12.5"
 
 [[override]]
   name = "k8s.io/code-generator"
-  version = "kubernetes-1.11.3"
+  version = "kubernetes-1.12.5"
 
 [[override]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.11.3"
+  version = "kubernetes-1.12.5"
 
 [[override]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.11.3"
+  version = "kubernetes-1.12.5"
 
 [[override]]
   name = "cloud.google.com/go"
   revision = "90f2606161ee6a14efe2ca79fc05ac2b8efe250b"
 
 [[constraint]]
   name = "github.com/knative/serving"
-  version = "v0.2.1"
+  version = "v0.3.0"
```

Comment:
I'm guessing this is 0.3 instead of 0.4 because eventing also uses 0.3

https://github.com/knative/eventing/blob/a99a1c259e6b9dafe1b3751b1660243a3662b150/Gopkg.toml#L104-L106

Reply:
yes, exactly.